presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/QueryBuilder.java

```diff
@@ -138,6 +140,9 @@ else if (typeAndValue.getType().equals(TinyintType.TINYINT)) {
             else if (typeAndValue.getType().equals(DoubleType.DOUBLE)) {
                 statement.setDouble(i + 1, (double) typeAndValue.getValue());
             }
+            else if (typeAndValue.getType().equals(RealType.REAL)) {
+                statement.setFloat(i + 1, intBitsToFloat((int) ((long) typeAndValue.getValue())));
```

Comment:
yeah but don't we use long for others too, I remember this double casting being there in the PR that added support for tinyint etc in JDBC connector and the reviewer suggested to use Number and then byteValue, we can use intValue from there